packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -144,10 +144,12 @@ class ExpansionPanelList extends StatelessWidget {
               curve: Curves.fastOutSlowIn,
               margin: _isChildExpanded(index) ? kExpandedEdgeInsets : EdgeInsets.zero,
               child: new SizedBox(
```

Comment:
I don't think the SizedBox is needed anymore

Reply:
Fixed